lib/activerecord-import/adapters/abstract_adapter.rb

```diff
@@ -16,7 +16,7 @@ def insert_many( sql, values, *args ) # :nodoc:
       sql2insert = base_sql + values.join( ',' ) + post_sql
       insert( sql2insert, *args )
 
-      number_of_inserts
+      [number_of_inserts,[]]
```

Comment:
Actually, ignore the above comment here. I'm more interested in the public interface that users consume, less so about internal library interfaces. Although I wouldn't be opposed if they all allowed for easier extension or addition.